azext_iot/central/params.py

```diff
@@ -386,3 +384,80 @@ def load_central_arguments(self, _):
             options_list=["--description", "--desc"],
             help="Detailed description of the job.",
         )
+
+    with self.argument_context("iot central query") as context:
+        context.argument(
+            "query_string",
+            options_list=["--query-string", "--qs"],
+            help="Query clause to retrieve telemetry or property data.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            choices=CaseInsensitiveList([ApiVersion.v1_1_preview.value]),
+            default=ApiVersion.v1_1_preview.value,
+            help="The API version for the requested operation.",
+        )
+
+    with self.argument_context("iot central export destination") as context:
+        context.argument(
+            "destination_id",
+            options_list=["--dest-id"],
+            help="Unique identifier for the destination.",
```

Comment:
Same as the command help, the arg help could include 'export' to be more clear
`Unique identifier for the destination.` -> `Unique identifier for the export destination.`

Reply:
Fixed